obspy/signal/headers.py

```diff
@@ -147,6 +147,14 @@
     C.c_int]
 clibsignal.calculate_kernel.restype = None
 
+clibsignal.aic_simple.argtypes = [
+    np.ctypeslib.ndpointer(dtype=np.float64, ndim=1,
+                           flags='C_CONTIGUOUS'),
+    np.ctypeslib.ndpointer(dtype=np.float64, ndim=1,
+                           flags='C_CONTIGUOUS'),
+    C.c_uint32]
```

Comment:
As in the other ones, you should comment the arg names here.

Reply:
Again, only the minority of your functions take comments in the header file. That's why I missed a need for them.
Added.
Another divergence of the header styles lies in the return type if it's void. Both `None` and `C.c_void_p` are valid. I was unsure which one I should use. Something for you to decide in a separate PR for all those functions.